### PR TITLE
Added json_body example.

### PR DESCRIPTION
**Pull request by klemens-morgenstern**

### CODE REVIEW

File: example/http/client/Jamfile [hunk 1]
```diff
@@ -16,3 +16,5 @@ build-project sync ;
 build-project async-ssl ;
 build-project coro-ssl ;
 build-project sync-ssl ;
+
+build-project body ;
```
Comment (vinniefalco) [inline example/http/client/Jamfile:20]: json-body ?

Comment (klemens-morgenstern) [reply example/http/client/Jamfile:20]: I named it body in case we get more body examples.

**Comment by klemens-morgenstern** [comment]
@vinniefalco waiting for your approval.

**Comment by vinniefalco** [comment]
Commit messages should not describe a past event but should instead a sentence using a form of the verb to-be. For example: "streaming JSON body is an example"

**Comment by vinniefalco** [comment]
Seems ok

**Pull request merged**